tests/test_deterministic_behaviour.py

```diff
@@ -65,22 +65,8 @@ def test_public_transport_travel_time_matrices(
         intermediate_results[transport_modes].append(travel_times)
 
         for matrix_a, matrix_b in pairwise(intermediate_results[transport_modes]):
-            m_a = matrix_a.set_index(["from_id", "to_id"])
-            m_b = matrix_b.set_index(["from_id", "to_id"])
-            m_a["difference"] = m_a.travel_time - m_b.travel_time
-            m_a["diff_rel"] = m_a.difference / m_a.travel_time
-            print(
-                "-----",
-                transport_modes,
-                # m_a[m_a.difference != 0],
-                m_a.difference.describe(),
-                m_a.diff_rel.describe(),
-                # m_a[(m_a.difference >= 3) | (m_a.difference <= -3)],
-            )
-
             pandas.testing.assert_frame_equal(
                 matrix_a,
                 matrix_b,
                 check_like=True,
-                atol=4,  # tolerance between runs, in minutes
             )
```

Comment:
And this removes the tolerance when comparing two consecutive runs of the same analysis: the resulting values now have to be 100% identical